packages/eslint-plugin-turbo/__fixtures__/framework-inference/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "framework-inference",
```

Comment:
We should (if we don't already) add a test for a package with multiple frameworks. This _could_ come up in a shared UI lib for example

Reply:
sounds great: added a kitchen-sink test here ce55b7d93e7169b5b1379948f2a1001f4e0d8f72